pkg/bundle/bundle.go

```diff
@@ -61,6 +61,9 @@ type Options struct {
 
 	// FilterExpiredCerts controls if expired certificates are filtered from the bundle.
 	FilterExpiredCerts bool
+
+	// FilterExpiredCerts controls if duplicate certificates are filtered from the bundle.
+	FilterDuplicateCerts bool
```

Comment:
Could it make sense to "reverse" the naming here so that the option can default to `false`? Or even remove the option and always deduplicate? Why would you ever chose to NOT deduplicate certs?

Reply:
You are right, there is no any reason to disable deduplication. I'm gonna remove this option.